src/data/pages/applications/major.js

```diff
@@ -114,7 +114,7 @@ const majorApplications = [
     name: 'Tetragon',
     logoName: 'tetragonLogo',
     title: 'eBPF-based Security Observability & Runtime Enforcement',
-    description: `Tetragon provides eBPF-based transparent security observability
+    description: `<a href="https://tetragon.io" style="text-decoration: underline;">Tetragon</a> provides eBPF-based transparent security observability
     combined with real-time runtime enforcement. The deep visibility is
```

Comment:
Please don't add links into the description sections here and above.

We do have the urls field:

```
    urls: [
      { label: 'GitHub', url: 'https://github.com/cilium/cilium' },
      { label: 'Website', url: 'https://cilium.io' },
    ],
```